app/screens/home/search/results/header.tsx

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {useCallback} from 'react';
+import {useIntl} from 'react-intl';
+import {Text, View} from 'react-native';
+import Button from 'react-native-button';
+
+import {useTheme} from '@context/theme';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+type Props = {
+    onToggle: (val: boolean) => void;
+    showMessages: boolean;
+    numberFiles: number;
+    numberMessages: number;
+}
+const Header = ({onToggle, showMessages, numberFiles, numberMessages}: Props) => {
```

Comment:
I think it is easier to keep track of the tab press and `showMessages`.   The selected `tab` press is not turning blue.  I made some minor refactoring in this file. 
[refactoring.txt](https://github.com/mattermost/mattermost-mobile/files/8706004/refactoring.txt)
.